server/test/api/account.js

```diff
@@ -0,0 +1,105 @@
+/*global describe, it, beforeEach, process*/
+
+var chai = require('chai');
+var chaiHttp = require('chai-http');
+var expect = chai.expect;
+chai.use(chaiHttp);
+
+if (!global.Promise) {
+  var q = require('q');
+  chai.request.addPromises(q.Promise);
+}
+
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = 0;
+
+var url = 'https://localhost:8080';
+var user = { username : 'superuser', password : 'superuser', project: 1};
+
+describe('The /account API endpoint', function () {
+  var agent = chai.request.agent(url);
+  var new_account = {
+    id : 1000,
+    account_type_id : 1,
+    enterprise_id : 1,
+    account_number : 4000400,
+    account_txt : 'Account for integration test',
+    parent : 0,
+    locked : 0,
+    cc_id : null,
+    pc_id : null,
+    classe : 4,
+    is_asset : 0,
+    reference_id : null,
+    is_brut_link : 0,
+    is_used_budget : 0,
+    is_charge : 0,
+    is_title : 0
+  };
+
+  var deletable_account = {
+    id : 3636
+  };
+
+  var fecthable_account = {
+    id : 3626
+  };
+
+  // throw errors
+  function handler(err) { throw err; }
+
+  // login before each request
+  beforeEach(function () {
+    return agent
+      .post('/login')
+      .send(user);
+  });
+
+  it(' A GET /accounts returns a list of accounts', function () {
```

Comment:
This description should probably mention that you are using a query string.  What about a test for when there are no query strings present?